packages/core-ethereum/crates/core-ethereum-db/src/db.rs

```diff
@@ -479,8 +510,26 @@ impl<T: AsyncKVStorage<Key = Box<[u8]>, Value = Box<[u8]>> + Clone> HoprCoreEthe
         //utils_log::debug!("DB: update_channel_and_snapshot channel_id: {}", channel_id);
         let channel_key = utils_db::db::Key::new_with_prefix(channel_id, CHANNEL_PREFIX)?;
         let snapshot_key = utils_db::db::Key::new_from_str(LATEST_CONFIRMED_SNAPSHOT_KEY)?;
+        let unrealized_balance_key =
+            utils_db::db::Key::new_with_prefix(channel_id, TICKETS_CHANNEL_UNREALIZED_BALANCE_PREFIX)?;
+
+        let new_balance = {
```

Comment:
Which code paths are calling this and what does it do?

Reply:
On channel change, called by the indexer.